### PR TITLE
bulk-scan and fees-register kustmization changes

### PR DESCRIPTION
**Pull request by AnilDhawaleGitHub**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ X] No
```

### CODE REVIEW

**Review by hannah38 — APPROVED** [review]

File: k8s/aat/common-overlay/fees-pay/kustomization.yaml [hunk 1]
```diff
@@ -7,11 +7,15 @@ bases:
 - ../../../namespaces/fees-pay/ccpay-service-request-cpo-update-service/ccpay-service-request-cpo-update-service.yaml
 - ../../../namespaces/fees-pay/ccpay-cpo-callback-function/ccpay-cpo-callback-function.yaml
 - ../../../namespaces/all-namespaces/nonprod-role.yaml
+- ../../../namespaces/fees-pay/ccpay-bulkscanning-api/ccpay-bulkscanning-api.yaml
```
Comment (hannah38) [inline k8s/aat/common-overlay/fees-pay/kustomization.yaml:10]: Hey, inconsistent naming. Is it api or app?

**Review by hannah38 — CHANGES_REQUESTED** [review]

File: k8s/namespaces/fees-pay/ccpay-bulkscanning-api/ccpay-bulkscanning-api.yaml [hunk 2]
```diff
@@ -5,9 +5,6 @@ metadata:
   namespace: fees-pay
```
Comment (hannah38) [inline k8s/namespaces/fees-pay/ccpay-bulkscanning-api/ccpay-bulkscanning-api.yaml:5]: Can you remove this from here please, set from kustomize level

File: k8s/namespaces/fees-pay/fees-register-api/fees-register-api.yaml [hunk 3]
```diff
@@ -5,9 +5,6 @@ metadata:
   namespace: fees-pay
```
Comment (hannah38) [inline k8s/namespaces/fees-pay/fees-register-api/fees-register-api.yaml:5]: Can you remove this from here please, set from kustomize level

**Review by hannah38 — CHANGES_REQUESTED** [review]

File: k8s/namespaces/fees-pay/ccpay-bulkscanning-api/ccpay-bulkscanning-api.yaml (hunk 2 above)
Comment (AnilDhawaleGitHub) [reply k8s/namespaces/fees-pay/ccpay-bulkscanning-api/ccpay-bulkscanning-api.yaml]: have removed it from fees-register and bulk-scan as well

**Review by hannah38 — APPROVED** [review]

**Review by santu412 — APPROVED** [review]

**Pull request merged**